Tests/SwiftParserTest/translated/SwitchTests.swift

```diff
@@ -259,9 +238,7 @@ final class SwitchTests: XCTestCase {
       }
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 2: all statements inside a switch must be covered by a 'case' or 'default'
         DiagnosticSpec(message: "unexpected text 'x = 1' in switch case"),
```

Comment:
Maybe keep a TODO? It'd be nice to handle this as a missing case rather than unexpected text.

Reply:
I improved the diagnostic here.